app/renderer/index.js

```diff
@@ -3,17 +3,37 @@ import { render } from 'react-dom';
 import { Provider } from 'react-redux';
 import { Router, hashHistory } from 'react-router';
 import { syncHistoryWithStore } from 'react-router-redux';
+import electron from 'electron';
+import WrongFolder from './components/WrongFolder/WrongFolder';
 import routes from './routes';
 import configureStore from './store/configureStore';
 import './styles/app.global.css';
 import './styles/github-gist-theme.global.css';
 
+const {app} = electron.remote;
+const isDev = process.env.NODE_ENV === 'development';
 const store = configureStore();
 const history = syncHistoryWithStore(hashHistory, store);
 
+function shouldShowWrongFolderComponent () {
```

Comment:
should we consider operating systems different from Mac OS?

Reply:
Yes. My first commit didn't account for that and threw a 'not a function error' on the Windows build (via AppVeyor). So I added the check that the function exists which accounts for non Mac.

Reply:
> So I added the check that the function exists which accounts for non Mac.

Can you please show which line of this PR does this OS verification?

Reply:
It doesn't verify the OS explicitly. It verifies that `isInApplicationsFolder` is defined. It's not defined in non Mac operating systems which I discovered through failed AppVeyor test from previous commits.

Reply:
then it worth adding a comment